includes/class-edd-db-customer-meta.php

```diff
@@ -0,0 +1,144 @@
+<?php
+
+class EDD_DB_Customer_Meta extends EDD_DB {
+
+	/**
+	 * Get things started
+	 *
+	 * @access  public
+	 * @since   2.6
+	*/
+	public function __construct() {
+		global $wpdb;
+
+		$this->table_name  = $wpdb->prefix . 'edd_customermeta';
+		$this->primary_key = 'meta_id';
+		$this->version     = '1.0';
+
+		add_action( 'plugins_loaded', array( $this, 'register_table' ), 11 );
+
+	}
+
+	/**
+	 * Get table columns and data types
+	 *
+	 * @access  public
+	 * @since   1.7.18
+	*/
+	public function get_columns() {
+		return array(
+			'meta_id'     => '%d',
+			'customer_id' => '%d',
+			'meta_key'    => '%s',
+			'meta_value'  => '%s',
+		);
+	}
+
+	/**
+	 * Register the table with $wpdb so the metadata api can find it
+	 *
+	 * @access  public
+	 * @since   2.6
+	*/
+	public function register_table() {
+		global $wpdb;
+		$wpdb->customermeta = $this->table_name;
+	}
+
+	/**
+	 * Retrieve customer meta field for a customer.
+	 *
+	 * @param   int    $customer_id   Customer ID.
+	 * @param   string $meta_key      The meta key to retrieve.
+	 * @param   bool   $single        Whether to return a single value.
+	 * @return  mixed                 Will be an array if $single is false. Will be value of meta data field if $single is true.
+	 *
+	 * @access  public
+	 * @since   2.6
+	 */
+	public function get_meta( $customer_id = 0, $meta_key = '', $single = false ) {
```

Comment:
Should we make this `protected` so that the only way to get customer meta data is to call `EDD_Customer::get_meta()`? If we make them both public we could end up with two ways to get the same data.